docs/src/main/tut/core-concepts.md

```diff
@@ -310,11 +310,9 @@ object service {
 }
 ```
 
-The code might be explanatory by itself but let's review the different services one by one:
+The code might be explanatory by itself but, but `sayHello` would be our unary RPC, where only the `@rpc` annotation would be needed in this case.
```

Comment:
Suggestion:
Here `sayHello` is our unary RPC, where only the `@rpc` annotation is needed.